### ISSUE TITLE
Placeholder error

### ISSUE THREAD

**Issue by poslegm** [yandex/clickhouse-jdbc#180]
If I try to execute sql request like 
```sql
CREATE TABLE Doges(
  id String,
  name String,
  urls String -- TODO is this column necessary?
)
```
I will receive error message `SQL execution failed (Reason: Not all parameters binded (placeholder 1 is undefined))`.

The problem is in question mark in comment. I think that `?` in sql comments should not be determined as placeholders.

Library version `0.1.36`.

**Comment by serebrserg** [comment]
released in 0.1.38

**Comment by poslegm** [comment]
Thank you!
